server/game/cards/01_SOR/events/ForceThrow.ts

```diff
@@ -0,0 +1,35 @@
+import { log } from 'console';
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { TargetMode, WildcardZoneName, WildcardCardType, Trait, EventName } from '../../../core/Constants';
+
+export default class ForceThrow extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '1705806419',
+            internalName: 'force-throw',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Choose a player. That player discards a card from their hand',
+            targetResolver: {
+                mode: TargetMode.Player,
+                immediateEffect: AbilityHelper.immediateEffects.discardCardsFromOwnHand({ amount: 1 }),
+            },
+            then: (thenContext) => ({
+                title: 'If you control a Force unit, you may deal damage to a unit equal to the cost of the discarded card',
```

Comment:
This title will only appear if the condition is met, and we should include the name of the discarded card and the amount for clarity. So we can say maybe `Deal damage to a unit equal to the cost of ${ifYouDoContext.target.title} (${ifYouDoContext.events[0].card.printedCost} damage)`

For example, `Deal damage to a unit equal to the cost of Wampa (4 damage)`